packages/@sanity/base/src/change-indicators/overlay/Connector.css

```diff
@@ -0,0 +1,59 @@
+@import 'part:@sanity/base/theme/variables-style';
+
+.debugRect {
+  stroke: #ccc;
+  fill: none;
+  pointer-events: none;
+  stroke-linecap: round;
+}
+
+.path {
+  fill: none;
+  stroke: #ccc;
+  pointer-events: none;
+  stroke-linecap: round;
+  stroke-linejoin: round;
+}
+
+.focused {
+  stroke: var(--brand-primary);
```

Comment:
This variable should be var(--focus-color)

Reply:
Thanks! I've fixed all of these now.